frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -151,11 +153,11 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
     }
   });
 
-  //whether the associated buildings info is open
-  const [asscBuildingsOpen, setAsscBuildingsOpen] = React.useState<boolean>(false);
   //whether the general info is open
   const [generalInfoOpen, setGeneralInfoOpen] = React.useState<boolean>(true);
 
+  const isBuilding = popUpContext.propertyTypeID === PropertyTypes.BUILDING;
```

Comment:
retyping this check everywhere was getting annoying

Reply:
Yes, this is good practice.